src/store/watchList/operations/getWatchListOperation.js

```diff
@@ -0,0 +1,34 @@
+import { createLogic } from 'redux-logic'
+import { normalize } from 'normalizr'
+
+import { moviesListSchema } from 'Schemas'
+import { userIdSelector, sessionIdSelector } from 'Store/auth/selectors'
+import { storeData } from 'Store/data/actions'
+import { GET_MOVIES } from '../types'
+import { requestMoviesStart, requestMoviesSuccess, requestMoviesError } from '../actions'
+
+export const getWatchListOperation = createLogic({
+  type: GET_MOVIES,
+  latest: true,
+  async process({ action, axios, getState }, dispatch, done) {
+    const { params = {} } = action
+    params.session_id = sessionIdSelector(getState())
+    const url = `/account/${userIdSelector(getState())}/watchlist/movies`
```

Comment:
`getState()` is used twice. lets extract it to the variable

Reply:
✅